### PR TITLE
Clarify TensorBoard UX

### PR DESCRIPTION
**Pull request by winstonaws**
Added explanations from: https://github.com/aws/sagemaker-python-sdk/issues/198

### CODE REVIEW

File: sagemaker-python-sdk/tensorflow_resnet_cifar10_with_tensorboard/tensorflow_resnet_cifar10_with_tensorboard.ipynb [hunk 1]
```diff
@@ -136,11 +136,11 @@
     "\n",
     "The parameter **```run_tensorboard_locally=True```** will run **TensorBoard** in the machine that this notebook is running. Everytime a new checkpoint is created by the training job in the S3 bucket, **```fit```** will download the checkpoint to the temp folder that **TensorBoard** is pointing to.\n",
     "\n",
-    "When the **```fit```** method starts the training, it will log the port that **TensorBoard** is using to display the metrics. The default port is **6006**, but another port can be choosen depending on its availability. The port number will increase until finds an available port. After that the port number will printed in stdout.\n",
+    "**TensorBoard** is not started until the **```fit```** method is called. When **```fit```** starts the training, it will log the port that **TensorBoard** is using to display the metrics. The default port is **6006**, but another port can be choosen depending on its availability. The port number will increase until finds an available port. After that the port number will printed in stdout.\n",
```
Comment (jesterhazy) [inline sagemaker-python-sdk/tensorflow_resnet_cifar10_with_tensorboard/tensorflow_resnet_cifar10_with_tensorboard.ipynb]: chosen not choosen

Comment (jesterhazy) [reply sagemaker-python-sdk/tensorflow_resnet_cifar10_with_tensorboard/tensorflow_resnet_cifar10_with_tensorboard.ipynb]: depending on its availability -> if 6006 is in use

File: sagemaker-python-sdk/tensorflow_resnet_cifar10_with_tensorboard/tensorflow_resnet_cifar10_with_tensorboard.ipynb [hunk 2]
```diff
@@ -136,11 +136,11 @@
...
+    "You can access **TensorBoard** on your SageMaker notebook instance at: [proxy/6006/](/proxy/6006/)(TensorBoard will not work if forget to put the slash, '/', in end of the url). If running locally outside of a SageMaker notebook instance, instead use [http://localhost:6006](http://localhost:6006). If TensorBoard started on a different port, adjust these URLs to match.This example uses the optional hyperparameter **```throttle_secs```** to generate training evaluations more often, allowing to visualize **TensorBoard** scalar data faster. You can find the available optional hyperparameters [here](https://github.com/aws/sagemaker-python-sdk#optional-hyperparameters)."
```
Comment (jesterhazy) [inline sagemaker-python-sdk/tensorflow_resnet_cifar10_with_tensorboard/tensorflow_resnet_cifar10_with_tensorboard.ipynb]: match.This -> match. This

Comment (jesterhazy) [inline sagemaker-python-sdk/tensorflow_resnet_cifar10_with_tensorboard/tensorflow_resnet_cifar10_with_tensorboard.ipynb]: allowing to -> allowing you to

**Comment by mckev-amazon** [comment]
Closing this PR as there has been no activity since June.  Feel free to re-open if you decide to pick this one up again!

**Pull request closed without merging**